packages/editor/src/components/local-autosave-monitor/index.js

```diff
@@ -37,47 +41,17 @@ const hasSessionStorageSupport = once( () => {
 	}
 } );
 
-/**
- * Function returning a sessionStorage key to set or retrieve a given post's
- * automatic session backup.
- *
- * Keys are crucially prefixed with 'wp-autosave-' so that wp-login.php's
- * `loggedout` handler can clear sessionStorage of any user-private content.
- *
- * @see https://github.com/WordPress/wordpress-develop/blob/6dad32d2aed47e6c0cf2aee8410645f6d7aba6bd/src/wp-login.php#L103
- *
- * @param {string} postId  Post ID.
- * @return {string}        sessionStorage key
- */
-function postKey( postId ) {
-	return `wp-autosave-block-editor-post-${ postId }`;
-}
-
 /**
  * Custom hook which returns a callback function to be invoked when a local
  * autosave should occur.
  *
  * @return {Function} Callback function.
  */
 function useAutosaveCallback() {
```

Comment:
This refactoring is great, it also makes me think this particular hook is not necessarily needed anymore.

Reply:
Perhaps, though we would still need to use `requestIdleCallback` somewhere. And wouldn't it be strange to have it in the action creator?

Reply:
Or do you just mean inline the hook in the `LocalAutosaveMonitor` component?

Reply:
Yeah, I meant inline,  but I don't feel strongly tbh